src/dashboard/src/components/helpers.py

```diff
@@ -189,28 +189,6 @@ def set_setting(setting, value=''):
     setting_data.save()
 
 
-def get_client_config_value(field):
-    clientConfigFilePath = '/etc/archivematica/MCPClient/clientConfig.conf'
-    config = ConfigParser.SafeConfigParser()
-    config.read(clientConfigFilePath)
-
-    try:
-        return config.get('MCPClient', field)
-    except:
-        return ''
-
-
-def get_server_config_value(field):
-    clientConfigFilePath = '/etc/archivematica/MCPServer/serverConfig.conf'
-    config = ConfigParser.SafeConfigParser()
-    config.read(clientConfigFilePath)
-
-    try:
-        return config.get('MCPServer', field)
-    except:
-        return ''
-
-
```

Comment:
This is where everything started! The Dashboard reading config from everywhere :(